eoscostarica/guide.eoscostarica.io
# docs(common):  Updated readme ENG #370

Changed logo for it to be visible in dark mode, updated developers guide screenshot, and removed version note.

###  Updated readme #370

### What does this PR do?

- resolves #370 and #485 

### Steps to test
1. Open guides readme, see new screenshot, updated logo, and updated prerequisites.

#### CheckList
